test/view/delimit.cpp

```diff
@@ -27,7 +27,7 @@ int main()
     ::models<concepts::View>(aux::copy(rng0));
     ::models_not<concepts::BoundedView>(aux::copy(rng0));
     ::models<concepts::RandomAccessIterator>(rng0.begin());
-    CONCEPT_ASSERT(RandomAccessView<delimit_view<std::vector<int>, int>>());
+    CONCEPT_ASSERT(RandomAccessView<delimit_view<std::vector<int> &, int>>());
```

Comment:
This is odd. The views adaptor class templates all expect to be parameterized over views. A reference to a container is not a view. What's up?

Reply:
IIRC, this is leakage from a failed experiment in which I hacked `view::all(ContiguousContainer&)` to return `span`. I clearly need to go through this patch line-by-line. (This patch is several months old, I only recently found time to polish it up and workaround GCC5 trying to instantiate `declval` in C++11 mode as it loves to do. I *thought* I had left the branch clean but that is obviously not the case.)